test-app/src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstemplatepackagename/integration/HmppsQueueResourceTest.kt

```diff
@@ -66,9 +66,9 @@ class HmppsQueueResourceTest : IntegrationTestBase() {
       val event2 = HmppsEvent("id2", "test.type", "message4")
       val message1 = Message(gsonString(event1), "message-id1", MessageAttributes(EventType("test.type", "String")))
       val message2 = Message(gsonString(event2), "message-id2", MessageAttributes(EventType("test.type", "String")))
-      inboundSqsDlqClient.sendMessage(inboundDlqUrl, gsonString(message1))
-      inboundSqsDlqClient.sendMessage(inboundDlqUrl, gsonString(message2))
-      await untilCallTo { inboundSqsDlqClient.countMessagesOnQueue(inboundDlqUrl) } matches { it == 2 }
+      outboundTestSqsClient!!.sendMessage(inboundDlqUrl, gsonString(message1))
+      outboundTestSqsClient!!.sendMessage(inboundDlqUrl, gsonString(message2))
+      await untilCallTo { outboundTestSqsClient!!.countMessagesOnQueue(inboundDlqUrl!!) } matches { it == 2 }
```

Comment:
I don't understand the change to outboundTestSqsClient... also too many bangbangs again, smells funny.  Might be better to cast it to the non-nullable type if we know it is never null - see comment in `IntegrationTestBase`

Reply:
Yes - I have just done that locally - for our current tests for that exact reason